### ISSUE TITLE
Win 7

### ISSUE THREAD

**Issue by harryray33** [donoh5/PasteIt#1]
Does this work with Win 7?

Thanks

**Comment by donoh5** [comment]
No, unfortunately, the software does not work on Windows 7 in my current environment due to a code signing issue. The operating system does not permit installation. I will update this thread once I obtain the necessary CA certificate and ensure compatibility with Windows 7 and 8. Thanks.

**Comment by harryray33** [comment]
Great, thank you...
